lib/DDG/Spice/QuoteOfTheDay.pm

```diff
@@ -0,0 +1,30 @@
+package DDG::Spice::QuoteOfTheDay;
+
+# ABSTRACT: Displays the quote for the day using Quotes REST API by TheySaidSo
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
```

Comment:
Lets get rid of the introductory comments - they're no longer needed (there's quite a few!)